python/core/qgsvectorlayereditbuffer.sip

```diff
@@ -148,4 +148,16 @@ class QgsVectorLayerEditBuffer : QObject
     void updateAttributeMapIndex( QgsAttributeMap& attrs, int index, int offset ) const;
 
     void updateLayerFields();
+
+    /** Apply geom modification basing on provider geometry type.
+     * Geom is modified only if successful convertion is possibile.
+     * In case of conversion a message is sent to the log.
+     * @param geometry pointer to the geometry that should be adapted to provider
+     * @return bool true if success.
+     *  Success means that input geometry is changed because conversion is applied or
+     *  geometry is untouched if geometry conversion is not necessary.
+     *  Fail if conversion is not possible and geometry is untuched.
```

Comment:
Typos:
**False** if conversion is not possible and geometry is unt**o**uched.

Can you also change the "Geom" strings to "Geometry", while at it?

Thanks